From 97e64eb0313ea62735235d305dcc57ea2da3f92b Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 18 Apr 2020 18:11:41 +0300
Subject: [PATCH 126/136] UninstallAlertDialogFragment: Catch
 IllegalArgumentException

  * from
  https://github.com/PixelExperience/frameworks_base/commit/1eb16de634eb52376ccce904e2c7f860c19cf0d1

	modified:   packages/PackageInstaller/src/com/android/packageinstaller/handheld/UninstallAlertDialogFragment.java

Change-Id: I2b339309df0ea324d206a8e31a5f5a37bfc6bff7
---
 .../packageinstaller/handheld/UninstallAlertDialogFragment.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/PackageInstaller/src/com/android/packageinstaller/handheld/UninstallAlertDialogFragment.java b/packages/PackageInstaller/src/com/android/packageinstaller/handheld/UninstallAlertDialogFragment.java
index 0a37cc6cb6c..30b5317d14d 100644
--- a/packages/PackageInstaller/src/com/android/packageinstaller/handheld/UninstallAlertDialogFragment.java
+++ b/packages/PackageInstaller/src/com/android/packageinstaller/handheld/UninstallAlertDialogFragment.java
@@ -77,7 +77,7 @@ public class UninstallAlertDialogFragment extends DialogFragment implements
             try {
                 stats = storageStatsManager.queryStatsForPackage(convert(volumes.get(i).getUuid()),
                         pkg, user);
-            } catch (PackageManager.NameNotFoundException | IOException e) {
+            } catch (PackageManager.NameNotFoundException | IOException | IllegalArgumentException e) {
                 Log.e(LOG_TAG, "Cannot determine amount of app data for " + pkg + " on "
                         + volumes.get(i) + " (user " + user + ")", e);
                 continue;
-- 
2.25.1

